src/cosmoz-toggle.ts

```diff
@@ -0,0 +1,80 @@
+import { html } from 'lit-html'; // eslint-disable-line object-curly-newline
+import { component } from 'haunted';
+
+export const Toggle = ({ label }) => {
+	console.log(label);
+	const changeHandler = () => {
+		const changeEvent = new CustomEvent('change');
+		dispatchEvent(changeEvent);
```

Comment:
call also a onChange function from props so that we have 2 ways to handle the change via event and callback.

Reply:
I am a bit unsure about TypeScript syntax here, I'll have to go through that documentation as well

Reply:
Also use `useCallback` to define the handler.

Reply:
@cristinecula I did insert the onChange call in a useCallback. I'm just a bit uncertain about the syntax/if it is the right function, but its call should be keptat every rendering...?

Reply:
That is not how you use `useCallback`. Read about it here: https://hauntedhooks.netlify.app/docs/hooks/usecallback/